common/src/main/java/org/opensearch/sdk/DeleteDataObjectRequest.java

```diff
@@ -73,12 +81,17 @@ public Builder id(String id) {
             return this;
         }
 
+        public Builder tenantId(String tenantId) {
+            this.tenantId = tenantId;
+            return this;
+        }
```

Comment:
I like how we're demonstrating how easy it is to add to these request objects to increase functionality.